docs_espressif/zh_CN/settings.rst

```diff
@@ -0,0 +1,289 @@
+      - git 可执行文件的路径
+    * - idf.gitPathWin
+      - Windows 系统中 git 可执行文件的路径
+    * - idf.enableCCache
+      - 在构建任务中启用 CCache（确保 CCache 在 PATH 中）
+    * - idf.enableIdfComponentManager
+      - 在构建命令中启用 IDF 组件管理器
+    * - idf.espIdfPath
+      - ESP-IDF 框架的位置路径 (IDF_PATH)
+    * - idf.espIdfPathWin
+      - Windows 系统中 ESP-IDF 框架的位置路径 (IDF_PATH)
+    * - idf.ninjaArgs
+      - Ninja 构建任务的参数
+    * - idf.pythonInstallPath
+      - 用于构建 ESP-IDF Python 虚拟环境的系统 Python 绝对路径
+    * - idf.toolsPath
+      - ESP-IDF 工具的位置路径 (IDF_TOOLS_PATH)
+    * - idf.toolsPathWin
+      - Windows 系统中 ESP-IDF 工具的位置路径 (IDF_TOOLS_PATH)
+
+扩展将按照以下方式使用上述设置：
+
+1. **idf.customExtraVars** 用于存储自定义环境变量，例如 OPENOCD_SCRIPTS，用于指定启动 OpenOCD 服务器时所需脚本文件的目录路径。这些变量会加载到扩展命令的进程环境变量中，优先使用扩展变量，如果没有，则扩展命令会尝试使用系统 PATH 中已有的设置。**该配置项不会改变 VS Code 之外的系统环境。**
+2. **idf.espIdfPath**（Windows 系统中为 **idf.espIdfPathWin**）用于在扩展中存储 ESP-IDF 目录路径。如果该值已配置，则 VS Code 进程中原有的 IDF_PATH 会被覆盖。**该配置项不会改变 VS Code 之外的系统环境。** 此外，扩展使用 **idf.espIdfPath**，结合 **idf.toolsPath** 和 **idf.pythonInstallPath**，来确定要添加到环境变量 PATH 中的 ESP-IDF 工具路径和 Python 虚拟环境路径。
+3. **idf.pythonInstallPath** 是系统 Python 的绝对路径，基于 **idf.toolsPath** 和 **idf.espIdfPath** 来生成 ESP-IDF Python 虚拟环境路径。创建虚拟环境后，ESP-IDF 的 Python 包将在该环境中安装和使用。
+4. **idf.gitPath**（Windows 系统中为 **idf.gitPathWin**）在扩展中用于克隆 ESP-IDF master 版本及其他支持的框架，如 ESP-ADF、ESP-MDF 和 Arduino-ESP32。
+5. **idf.toolsPath**（Windows 系统中为 **idf.toolsPathWin**）用于结合 **idf.toolsPath** 和 **idf.pythonInstallPath** 来确定要添加到环境变量 PATH 中的 ESP-IDF 工具路径和 Python 虚拟环境路径。
+
+.. note::
+
+    在 VS Code 扩展中，系统 PATH 及其他环境变量是无法修改的。在扩展的所有任务和子进程中，本扩展使用的是修改后的进程环境，且不应影响其他系统进程及扩展。请检查 **idf.customExtraVars** 的配置，以防与其他扩展发生冲突。
+
+开发板/芯片相关设置
+--------------------
+
+以下是针对 ESP32 芯片/开发板的配置项。
+
+.. list-table::
+    :widths: 25 75
+    :header-rows: 1
+
+    * - 设置
+      - 描述
+    * - **idf.flashBaudRate**
+      - 烧录速率
+    * - **idf.monitorBaudRate**
+      - 监视器速率（默认留空，通过 SDKConfig 中的 ``CONFIG_ESP_CONSOLE_UART_BAUDRATE`` 进行配置）
+    * - **idf.openOcdConfigs**
+      - OpenOCD 配置文件路径，相对路径基于 ``OPENOCD_SCRIPTS`` 文件夹
+    * - **idf.openOcdLaunchArgs**
+      - OpenOCD 启动参数，位于 ``idf.openOcdDebugLevel`` 和 ``idf.openOcdConfigs`` 之前
+    * - **idf.openOcdDebugLevel**
+      - 设置 OpenOCD 调试级别 (0-4)，默认值为 2
+    * - **idf.port**
+      - 选择的设备端口路径
+    * - **idf.portWin**
+      - Windows 系统中选择的设备端口路径
+    * - **idf.enableSerialPortChipIdRequest**
+      - 启用芯片 ID 检测功能，并在串口选择列表中显示 ID
+    * - **idf.useSerialPortVendorProductFilter**
+      - 启用 ``idf.usbSerialPortFilters`` 列表以过滤串口设备列表
+    * - **idf.usbSerialPortFilters**
+      - 用于过滤已知乐鑫设备的 USB productID 和 vendorID 列表
+    * - **openocd.jtag.command.force_unix_path_separator**
+      - 强制在 Windows 操作系统中使用 ``/`` 作为路径分隔符，而不是 ``\\``
+    * - **idf.svdFilePath**
+      - SVD 文件的绝对路径，用于解析芯片在调试器中的外设树视图
+
+扩展将按照以下方式使用上述设置：
+
+1. **idf.flashBaudRate** 是用于 **ESP-IDF：烧录项目** 命令和 `Debugging <https://github.com/espressif/vscode-esp-idf-extension/blob/master/docs/DEBUGGING.md>`_ 的烧录速率。
```

Comment:
这里和之前的想法一样，个人不认为也要把 ESP-IDF: Flash your Project 命令翻译成中文，而且这里的  调试 文档是不是以后还要附上中文翻译文档链接

Reply:
Hi @zztiger123 The author has updated this link to point to a [website page](https://docs.espressif.com/projects/vscode-esp-idf-extension/en/latest/debugproject.html). Currently, this page is only available in English. If we replace "en" in the link with "zh_CN", it will turn 404. So, do you think it would be better to edit the link now in case we forget to revise it later when CN page is available, or should we use the same link as in the EN doc?

Reply:
We can use the same link as in the EN doc now.